components/grid/templates/filter.md

```diff
@@ -43,7 +43,7 @@ The example below shows a custom filter that:
 * Shows how you can store a reference to the context or use it inline in the template.
 * Showcases building a filter descriptor with two filters and sample logic that always filters the data even if one of the inputs is empty.
 
-You can find more examples in the [Live Demo: Custom Filter Row](https://demos.telerik.com/blazor-ui/grid/custom-filter-row) that is available in your local installation under the `demos` folder.
+You can find more examples in the [Live Demo: Custom Filter Row](https://demos.telerik.com/blazor-ui/grid/custom-filter-row) that is available in your local installation under the `demos` folder. Also check [How to implement a filter operator dropdown in a Filter Row Template]({%slug grid-kb-filter-operator-dropdown%}) in order to mimic the default filter row UI.
```

Comment:
Ideally, we would be using consistent spelling for the term. [Merriam Webster](https://www.merriam-webster.com/dictionary/drop-down) recommends "drop-down" or "dropdown" (as a variant), but not "drop down" (see the KB) for the noun. For adjectives, it's usually "drop-down". However, the [Microsoft Manual of Style](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/dropdown) goes with "dropdown", but recommends not using it a standalone noun.

I would recommend going with "dropdown" for the time being, since we're more attuned to what MS are doing. However, you might want to be more precise and use "dropdown menu", "dropdown list", or whatever fits best. Going with DropDown (as in a component name) is also acceptable.